app/routes.ts

```diff
@@ -11,6 +11,7 @@ export default [
 
   layout("./layouts/default-layout.tsx", [
     // verfahren
+    route("verfahrens-uebersicht", "./routes/verfahrensUebersicht.tsx"),
```

Comment:
I suggest naming the url "verfahren" instead of "verfahrens-uebersicht". It's the index page.